content/en/tracing/setup/dotnet-core.md

```diff
@@ -211,6 +211,7 @@ The .NET Tracer can instrument the following libraries automatically:
 | Redis (ServiceStack client)    | `ServiceStack.Redis` 4.0.48+                                            | `ServiceStackRedis`  |
 | Elasticsearch                  | `Elasticsearch.Net` 5.3.0+                                              | `ElasticsearchNet`   |
 | MongoDB                        | `MongoDB.Driver.Core` 2.1.0+                                            | `MongoDb`            |
+| PostgreSQL                     | `Npgsql` 4.0+                                                           | `Postgres`           |
```

Comment:
The integration name (last column) is `AdoNet`. [See here.](https://github.com/DataDog/dd-trace-dotnet/blob/7fcc12ea86b59f5bc7027e060bd3d9ae37262ede/src/Datadog.Trace.ClrProfiler.Managed/Integrations/AdoNet/NpgsqlCommandIntegration.cs#L16)

Reply:
Thanks -- didn't know that that was the intention of the column.

Reply:
Users need to know the integration name to change settings like `DD_<INTEGRATION>_ENABLED` or `DD_<INTEGRATION>_ANALYTICS_ENABLED`. In code, they access integration settings through `TracerSettings.Integrations["<INTEGRATION>"]`.